src/Service/PromotionsService.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace BBC\ProgrammesPagesService\Service;
+
+use BBC\ProgrammesPagesService\Cache\CacheInterface;
+use BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\PromotionRepository;
+use BBC\ProgrammesPagesService\Domain\Entity\Programme;
+use BBC\ProgrammesPagesService\Domain\Exception\DataNotFetchedException;
+use BBC\ProgrammesPagesService\Mapper\ProgrammesDbToDomain\PromotionMapper;
+use DateTimeImmutable;
+
+class PromotionsService extends AbstractService
+{
+    /** @var PromotionRepository */
+    protected $repository;
+
+    public function __construct(
+        PromotionRepository $repository,
+        PromotionMapper $mapper,
+        CacheInterface $cache
+    ) {
+        parent::__construct($repository, $mapper, $cache);
+    }
+
+    /**
+     * @return Promotion[]
+     * @throws DataNotFetchedException if we get a promotion but there is nothing being
+     *     promoted (an image or core entity) associated to the promotion
+     */
+    public function findActivePromotionsByProgramme(
+        Programme $programme,
+        DateTimeImmutable $dateTime,
```

Comment:
Why pass `ApplicationTime::getTime()` in to the repository method but not this method? Surely it should be both or neither?